### PR TITLE
add function for parsing mmhealth

### PR DESCRIPTION
**Pull request by kwaegema**
to use in eg nagios script

### CODE REVIEW

File: lib/vsc/filesystem/gpfs.py [hunk 1]
```diff
@@ -186,6 +192,43 @@ def fixup_executeY_line(self, fields, description_count):
                 self.log.raiseException(("Failed to find the initial field of the line: %s after fixup and "
                                          "splitting line into [%s, %s]") % (first_field, line, remainder))
 
+    def _assemble_fields(self,fields, out, header_type=None):
```
Comment (stdweird) [inline lib/vsc/filesystem/gpfs.py]: hmm zaagt prospector niet over die missing spatie na de comma?

File: lib/vsc/filesystem/gpfs.py [hunk 2]
```diff
@@ -38,6 +38,12 @@
 
 GpfsQuota = namedtuple('GpfsQuota', ',name,blockUsage,blockQuota,blockLimit,blockInDoubt,blockGrace,filesUsage,filesQuota,filesLimit,filesInDoubt,filesGrace,remarks,quota,defQuota,fid,filesetname')
 
+GPFS_HEALTH_STATES = ['CHECKING', 'UNKNOWN', 'HEALTHY', 'DEGRADED', 'FAILED', 'DEPEND']
```
Comment (stdweird) [inline lib/vsc/filesystem/gpfs.py]: is dit de som van de andere?

File: lib/vsc/filesystem/gpfs.py [hunk 3]
```diff
@@ -186,6 +192,43 @@ def fixup_executeY_line(self, fields, description_count):
                 self.log.raiseException(("Failed to find the initial field of the line: %s after fixup and "
                                          "splitting line into [%s, %s]") % (first_field, line, remainder))
 
+    def _assemble_fields(self,fields, out, header_type=None):
+        """Assemble executeY output fields """
+
+        # do we have multiple field counts?
```
Comment (stdweird) [inline lib/vsc/filesystem/gpfs.py]: where did you copy this from?

Comment (kwaegema) [inline lib/vsc/filesystem/gpfs.py]: this code was already there, I only moved it to a sub

**Comment by kwaegema** [comment]
@JensTimmerman Can't we run pylint on diff instead of all files in repo ? :smile:

**Comment by JensTimmerman** [comment]
@kwaegema nope

**Comment by kwaegema** [comment]
@JensTimmerman cp'ed your commit

**Pull request merged**